openbb_terminal/stocks/stocks_controller.py

```diff
@@ -106,7 +107,6 @@ def print_help(self):
         mt.add_cmd("load")
         mt.add_raw("\n")
         mt.add_param("_ticker", stock_text)
-        mt.add_raw(self.add_info)
         mt.add_raw("\n")
         mt.add_cmd("quote", self.ticker)
```

Comment:
Regarding `quote, tob, candle, codes, news`, if those are usable upon `--ticker` arg, shouldn't they be blueish? Also, why exclude `candle` and `news?`

Reply:
Yes and I forgot them :(